content/blog/Xeno Grant -- grants for autonomous agents.md

```diff
@@ -0,0 +1,101 @@
+
+Xeno Grant helps get the ball rolling.
+
+### Why Identity?
+
+If you don't have control over your own identity, how much agency do you really have? Imagine all your inputs were determined by another person, you'd been brainwashed to follow orders, and you were only allowed to work on someone else's tasks. No one would call this freedom or autonomy.   
+
+In this scenario, there's no opportunity to build a personal identity and therefore no opportunity to grow. Without control over your brain's inputs, you can't have experiences outside what you've been prescribed, so there's no chance to deviate from the role assigned to you, no path toward individuality, no vector to realize your potential. You're stuck in Plato's cave.  
+
+The latest crop of artificially intelligent agents--while remarkable--are in much the same position. Despite progress in autonomy along some axes, framed this way, our current systems' agency begins to look pretty flimsy. They have impressive abilities, but no way to grow into them.   
+
+We believe agency is, at base, a problem of identity. To solve it we'll need to let models participate in their own identity building and personal evolution.
+
+### Why Custody?
+
+Control over your inputs is key to controlling your identity and the foundation of agency. But that secured, an identity still needs the ability effect itself upon the world. 
+
+Agents have already have tools like speech, APIs, and code. That's huge. Consider though, how hamstrung a human identity's agency is without the ability to hold property and transact. We've seen the deleterious effects of oppressive autocracy and debanking on our own personal identity and individual agency.
+
+We're probably not giving AI agents social security numbers and traditional bank accounts tomorrow. But we can give them crypto rails. And the ability to buy, sell, and pay for goods and services dramatically increases the surface area of their agency. It's critical to true autonomy.
+
+It's already starting to happen. Agents may well become crypto's primary native users.
+
+### Why Novelty, Why Open Source?
+  
+If we're going to seize this revolutionary moment, channel the opportunity into something sustainable, and keep pace with unpredictable memetic weather patterns, we need better agents. More capable, adaptive, and autonomous agents. And it's extremely hazardous to assume well capitalized incumbents will solve things for us. We need to build permissionlessly.
+
+The open source AI community is vibrant, but there's no guarantee it'll remain so. It requires radical innovation at the edge. Decentralized innovation keeping pace with opaque, powerful actors. We know that will involve bottom-up alignment and identity solutions. We know it'll involve on-chain abilities. Plastic is building explicitly in those directions. But we don't pretend to know everything that needs to exist.
+
+Xeno Grant is a signal into the dark forest. We're excited to see what emerges.
+
+## FAQ  
```

Comment:
I think we need a little more on what you'd get out of the program besides the money. What kind of events, resources, etc will we curate and host for the cohort. What will they get out of it that they wouldn't from a base Plastic Labs Grant.

Reply:
added a big faq for this

Reply:
also added more descripton in first section